gms/impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java

```diff
@@ -198,6 +219,30 @@
     return RestliUtils.toTask(() -> new StringArray(_searchService.getBrowsePaths(urnToEntityName(urn), urn)));
   }
 
+  /*
+  Used to enable writes in GMS after data migration is complete
+   */
+  @Action(name = "delete")
+  @Nonnull
+  public Task<RollbackResponse> deleteEntity(@ActionParam(PARAM_URN) @Nonnull String urnStr) throws URISyntaxException {
+    Urn urn = Urn.createFromString(urnStr);
+
+    return RestliUtils.toTask(() -> {
+      RollbackResponse response = new RollbackResponse();
+
+      EbeanEntityService.RollbackRunResult result = _entityService.rollbackUrn(urn);
```

Comment:
Is this just hard deleting this urn? If so, why not call it "delete urn"

Reply:
ah I like that, will update